### DISCUSSION TITLE
Compiling Does Not Edit PDF Document In Window

### DISCUSSION THREAD

**Discussion by godowning** [texstudio-org/texstudio#3646] [Q&A]
Total beginner with TeXstudio, using existing document as template for new paper. 
After making edits to existing document (template) then compiling, the PDF document in the right window does not change. A new edited file is created in a text format. But the PDF does not change.

**Comment by godowning** [comment]
When I compile the Message reads:
Process started: pdflatex.exe -synctex=1 -interaction=nonstopmode "GSC 2624-0941_Alton&Downing_2021_Final Revisions_v1".tex
Process exited normally

**Reply by octaeder** [reply]
What happens when you press F5 (Build&View) or F7 (View)?

**Comment by godowning** [comment]
fn+F5: does the same as pressing Build and View. No change to the PDF document and a new text file is created. 
fn+F7: The only response I can see is that the text in the Message window is cleared. 
I'm running Windows 11.

**Comment by octaeder** [comment]
What happens when you copy the LaTeX code to clipboard, open a new editor with Ctrl+N and paste from clipboard into the editor and try F5?

**Comment by godowning** [comment]
It does not compile. Message: error_4_'aastex63.cls' not found. \usepackage

**Reply by octaeder** [reply]
looks like a mistake in the LaTeX code

**Comment by octaeder** [comment]
Create a new document with the main menu Wizards/Quick Start (simply confirm dialog with ok). Enter some simple text or math into the document environment and press F5.

**Comment by godowning** [comment]
Could it be that I don't have the aastex.cls classfile?? This is from the American Astronomical Society

**Reply by octaeder** [reply]
could be.

**Comment by godowning** [comment]
I did as you asked, created a new document and pressed F5. A PDF did appear in the right window with the test text.

**Comment by octaeder** [comment]
So it works in principle. Has your LaTeX distribution installed the necessary package? MiKTeX for example can be configured to load packages needed.

**Comment by godowning** [comment]
I didn't install the aastex63 and don't know how to do this. I'm looking at the AAS author guide now. Will have to do some reading in MiKTeX to learn how to download/install. I have downloaded the package but don't know how to move it over to MiKTeX/TeXstudio.

**Reply by octaeder** [reply]
You don't need to install it manually. Start MiKTeX Console (just type miktex in the windows start search field). Select `Settings` and see that `Always` is checked:

![image](https://github.com/texstudio-org/texstudio/assets/102688820/be83614a-5d13-46cb-9745-87d20016c964)

Select `Packages`, enter aastex into the search field and press the filter icon:

![image](https://github.com/texstudio-org/texstudio/assets/102688820/e103af64-76a1-4a60-816c-365210a6edfe)

You should see something like this:

![image](https://github.com/texstudio-org/texstudio/assets/102688820/700ec257-dd81-4eac-9dbc-5448644f84ab)

As you can see I have not installed the package. Use menu to install it if it won't be (always, s. above) installed by your compile:

![image](https://github.com/texstudio-org/texstudio/assets/102688820/93b24d8c-157f-4fc8-9dcd-016c5e62816e)

**Comment by octaeder** [comment]
You may find it usefull to know that txs main menu `Help` offers a package help dialog:

![image](https://github.com/texstudio-org/texstudio/assets/102688820/cd336fd0-fa04-46da-9910-638311403c3c)

**Comment by godowning** [comment]
O.K. I followed the instructions and loaded the package. Closed TeXstudio, restarted and it appears the problem persists. Could be simple operator error at this stage. 
Unfortunately I'm being called away from my desk now. THANK YOU FOR YOUR ASSISTANCE. I still have so much to learn if I am to complete this paper.

**Reply by octaeder** [reply]
You could check tex.stackexchange.com to address LaTeX issues.
